### PR TITLE
Upgrade to 20236 and update th default

### PR DESCRIPTION
**Pull request by jgsqware**
Towards https://github.com/giantswarm/giantswarm/issues/12974

Signed-off-by: Julien Garcia Gonzalez <julien@giantswarm.io>

The modification is aligned with the recommendation we got from AquaTeam

### CODE REVIEW

File: helm/aqua-app-server/templates/gate-deployment.yaml [hunk 1]
```diff
@@ -33,6 +33,10 @@ spec:
           value: "{{ .Release.Name }}-console-svc:443"
         - name: AQUA_WORKLOADS_INTEGRITY_CHECK
           value: {{ .Values.gate.integrityCheck | default "true" | quote }}
+        - name: AQUA_WORLOADS_PING_CHECK
+          value: {{ .Values.gate.pingCheck | default "true" | quote }}
+        - name: AQUA_DISCONNECTION_GRACE_PERIOD
+          value: {{ .Values.gate.disconnectionGracePeriod | default "true" | quote }}
```
Comment (glitchcrab) [inline helm/aqua-app-server/templates/gate-deployment.yaml:39]: is this actually a boolean variable? if it isn't then defaulting to true could break stuff.

**Comment by glitchcrab** [comment]
Seems OK to me, aside from the single comment above. Have we tested this with our test license?

File: helm/aqua-app-server/templates/gate-deployment.yaml (hunk 1 above)
Comment (jgsqware) [reply helm/aqua-app-server/templates/gate-deployment.yaml:39]: it's a info we got from them

Comment (jgsqware) [reply helm/aqua-app-server/templates/gate-deployment.yaml:39]: > Gateway - AQUA_DISCONNECTION_GRACE_PERIOD (boolean; true by default)

File: helm/aqua-app-enforcer/values.yaml [hunk 2]
```diff
@@ -28,17 +30,15 @@ gate:
...
-  {}
-  # limits:
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+  limits:
+    memory: 1500Mi
+  requests:
+    memory: 1500Mi
```
Comment (stoehdoi) [inline helm/aqua-app-enforcer/values.yaml:42]: Shouldn't we set it to 512Mi as this is the recommended minimum for the enforcer to run?

Comment (jgsqware) [reply helm/aqua-app-enforcer/values.yaml:42]: > Enforcer memory limit should be set to 'memory: 1500Mi' (the minimum requirement is 512Mi);

should and minimum requirement is not clear
 :)

**Pull request merged**